src/Tribe/Views/V2/Views/List_View.php

```diff
@@ -98,19 +98,34 @@ protected function get_past_url( $canonical = false, $page = 1 ) {
 		$date           = $this->context->get( 'event_date', $default_date );
 		$event_date_var = $default_date === $date ? '' : $date;
 
+
```

Comment:
Non-essential thing: extra line break here that we probably don't need. Also not a functional concern :-)